example/ios/LiveMarkdownExample-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```

Comment:
Do we need this file?

Reply:
Nope, [cbaebd4](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/cbaebd4853ef799b4b0f4de94d9568b0cd19f24a)